source/index.html.md

```diff
@@ -503,6 +504,72 @@ One of the reasons why SMART on FHIR is awesome is because of the interoperabili
+
+Field | Description
+--------- | -----------
+App Name | ```My amazing SMART Patient App``` Any name will do.
+SMART Launch URI | Leave this field blank since this is a standalone app.
+Redirect URI | ```https://<your-username>.github.io/smart-on-fhir-tutorial/example-smart-app/```
+App Type | ```Patient```
+FHIR Spec | ```dstu2_provider``` The latest spec version supported by Cerner.
+Authorized | ```Yes``` Authorized App will go through secured OAuth 2 login.
+Standard Scopes | These scopes are required to launch the SMART app.
+User Scopes | None
+Patient Scopes | Select the Patient and Observation scopes
+
+Click "Register" to complete the process. This will add the app to your account and create a client id for app authorization.
+
+The new OAuth 2 client id will be displayed in a banner at the top of the page and can be viewed at any time by clicking on the application icon to view more details.
+
+## Request Authorization
+
+The `launch-patient.html` file had been created for you already.  You'll need to update the client id with the new one you've just gotten. This file shows what your app will use to request authorization with the Authorization server.
+
+> launch-patient.html
+
+```html
+  <script>
+    FHIR.oauth2.authorize({
+      'client_id': '<enter your client id here>',
+      'scope':  'patient/Patient.read patient/Observation.read launch/patient online_access openid profile'
+    });
+  </script>
+```
+
+> Make sure to replace CLIENT_ID with the new client id provided in code Console. Notice that `launch/patient` is used here instead of `launch`.  This is because during a standalone launch the app does not have access to the patient in context and thereby it would need to request acces to the patient.
+
+## Launch a Standalone App
+
+Since this app is a standalone app, it does not need to be launched by the EHR (code Console). You can launch the app by copy/paste the URL below into your favorite browser.
+
+> Cerner's Sandbox Patient Access Endpoint:
+
+```
+https://fhir-myrecord.sandboxcerner.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca
+```
+
+The `iss` value in the query parameter represents the URL for our Sandbox Patient Access endpoint.  This value tells the app where to look for the `metadata` endpoint, which contains the authorization endpoints that the app needs to call.
+
+> Launch URL:
+
+```
+  https://<your-username>.github.io/smart-on-fhir-tutorial/example-smart-app/launch-patient.html?iss=https://fhir-myrecord.sandboxcerner.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca
+```
+
+Replace `<your-username>` with your GitHub's username. Then, enter the URL above into the browser, the browser should redirect to the login page. You can use patients' credentials listed in the first post in this <a href="https://groups.google.com/forum/#!topic/cerner-fhir-developers/edPUbVPIag0" target="_blank">discussion</a>.
```

Comment:
We should probably include a step telling them to commit their client id change before hitting this url on the gh-pages branch.

Reply:
good point